src/Views/Shared/MultipleFileUpload.cshtml

```diff
@@ -5,8 +5,8 @@
     var elements = Model.GenerateElementProperties();
     var baseCssClasses = "smbc-file-upload govuk-!-margin-bottom-5 govuk-!-display-block";
     elements.Add("class", !Model.IsValid
-        ? $"{baseCssClasses} govuk-input--error"
-        : baseCssClasses);
+        ? $"{baseCssClasses} govuk-input--error smbc-input--file-error "
+        : $"{baseCssClasses}");
```

Comment:
can remove string interpolation on else of ternary, as no other variables being appended.